index.html

```diff
@@ -3186,21 +3186,27 @@ <h3>Terms of Use</h3>
         </p>
 
         <p>
-The value of the <code>termsOfUse</code> <a>property</a> tells the
-<a>verifier</a> what actions it is required to perform (an <em>obligation</em>),
-not allowed to perform (a <em>prohibition</em>), or allowed to perform (a
-<em>permission</em>) if it is to accept the <a>verifiable credential</a> or
-<a>verifiable presentation</a>.
-        </p>
+Examples for the value of the <code>termsOfUse</code> <a>property</a> might be
+to tell the <a>verifier</a> any or all of the following:
+
+<ul><li>
+the procedures or policies that were used in issuing the <a>verifiable credential</a>,
+for example, by providing a pointer to the location where these procedures 
+or policies can be found, or naming the standard that defines them;
+</li>
+<lic
+the issuer's rules and policies that govern the presentation of this 
+<a>verifiable credential</a> to the <a>verifier</a>;
+</li>
+<li>
+the identity of the entity under whose authority the issuer issued this particular <a>verifiable credential</a>.
+</ul></li>
```

Comment:
done

Reply:
This was *not* done. *Parts* of it were manually applied, but parts were *not*. It is incredibly frustrating that this happens repeatedly. If you use GitHub's web interface, there are single-click buttons that will add such suggestions to a batch commit, or will merge each suggestion as its own commit, and then parts of suggestions would not be lost because you didn't notice them. The HTML markup is as important as the text it formats. A `<li>` must be closed before the `<ul>` in which it occurs. `</ul></li>` is incorrect. I am concerned that I now need to review all my previously suggested changes, letter by letter, to confirm that they have been accurately applied. This does not make me happy.